### DISCUSSION TITLE
New dual case (zero2/3A+) for RAD_zero_v01

### DISCUSSION THREAD

**Discussion by StmLord** [frntc/RAD#5] [General]
I have designed and uploaded a new case that can host a zero_v01 PCB with either a zero2 or 3A+ raspberries
It is only a 2-part thing and does not "need" a label as all texts are imprinted on the front part
https://www.thingiverse.com/thing:5893033

when I find few hours time I will make one for the sidekick64 project as it is quite close but all buttons should be soldered on the upper side in that case.

**Comment by frntc** [comment]
Nice concept!

I wonder, you would supply power for the Zero2 from the C64/C128 in this case?

**Reply by StmLord** [reply]
there are 2 holes on it, one on the right for the A3+ and one on the left for Z2. there are quite big so a small plug can be fully inserted.
For the Z2 the plug will be all the way inside the case of course, but I don't think its bad, because that way it keeps the micro usb connector a bit more safe, it will be easier to plug it on the zero with the case open and then close it. That could also give the chance to check for the jumper so there will be no disaster :D
